x-pack/plugins/infra/public/pages/metrics/metrics_explorer/components/chart.tsx

```diff
@@ -77,10 +77,7 @@ export const MetricsExplorerChart = ({
   const dateFormatter = useMemo(
     () =>
       series.rows.length > 0
-        ? niceTimeFormatter([
-            (first(series.rows) as any).timestamp,
-            (last(series.rows) as any).timestamp,
-          ])
+        ? niceTimeFormatter([first(series.rows)!.timestamp, last(series.rows)!.timestamp])
```

Comment:
Similar to an case above we could give `firstTimestamp` and `lastTimestamp` a name and explicitly check for their `undefined`ness.